apixu-weatherbot1.0.tcl

```diff
@@ -0,0 +1,498 @@
+#!usr/bin/env tclsh
+
+########################################################################################
+# Name              apixu-weatherbot1.0.tcl
+#
+# Author            Bruce Olivier <bolivierjr@gmail.com>
+#
+# Description       Uses the Apixu weather API to grab the data.
+#
+#                   Requires channel to be flagged with +weather-apixu
+#                       example: .chanset #chan +weather-apixu
+#
+#                   Commands:
+#                       .set 0-2 <location>
+#                          - can be 0 for Metric, 1 for Imperial, 2 for both.
+#                            i.e. ".set 2 New Orleans, LA"
+#                       .wz <location>
+#                          - Grabs current weather location. If no location is
+#                            given, will try to use users saved location
+#
+#                       .wzf <location>
+#                          - Same as .wz but provides a 5 day forecast
+#                       
+#                       Use the --help or -h flags with any of the commands to
+#                       get help with the commands and descriptions.
+#
+# Version           1.0 - Initial Release
```

Comment:
nit:  Consider [semver](https://semver.org/)

Reply:
oh, that's good to know. Thanks for the insight